dhis-2/dhis-api/src/main/java/org/hisp/dhis/option/OptionSet.java

```diff
@@ -122,6 +123,16 @@ public void removeAllOptions()
         options.clear();
     }
 
+    public void removeOption( Option option )
+    {
+        if ( options == null || CollectionUtils.isEmpty( options ) )
```

Comment:
Main reason to use a static `isEmpty` is that it allows for `null` so it does not need a `null` check before

Reply:
fixed